### ISSUE TITLE
Unable to create new rootless pods after using podman stop --all and podman rm --all

### ISSUE THREAD

**Issue by jdoss** [containers/podman#16162]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

I think I put my rootless podman into a bad state. I had a bunch of containers unrelated to my pods/containers I run on a daily basis on my workstation. I decided rather than indivudally clean everything up to do `podman stop --all` and `podman rm --all` and then relaunch my pods/containers that I wanted running via a bash shell script. After doing that I could no longer start any pods. 

```
$ podman pod ls
POD ID      NAME        STATUS      CREATED     INFRA ID    # OF CONTAINERS
$ podman pod create foo
Error: unable to create pod cgroup for pod 2e75d2c8edb0d15e52f814f6f11af9c4a68087e8342f453c80ecf83fb50f9ff6: creating cgroup user.slice/user-libpod_pod_2e75d2c8edb0d15e52f814f6f11af9c4a68087e8342f453c80ecf83fb50f9ff6.slice: Unit user-libpod_pod_2e75d2c8edb0d15e52f814f6f11af9c4a68087e8342f453c80ecf83fb50f9ff6.slice already exists.
```
I have logged out and back in and the issue continued, so I rebooted hoping it could clear up but it remains the same. 

**Steps to reproduce the issue:**

1. Have a bunch of pods with containers running inside them. 
2. `podman stop --all` and `podman rm --all
3. ???
4. See you can no longer create pods.

**Describe the results you received:**

```
$ podman pod ls
POD ID      NAME        STATUS      CREATED     INFRA ID    # OF CONTAINERS
$ podman pod create foo
Error: unable to create pod cgroup for pod 2e75d2c8edb0d15e52f814f6f11af9c4a68087e8342f453c80ecf83fb50f9ff6: creating cgroup user.slice/user-libpod_pod_2e75d2c8edb0d15e52f814f6f11af9c4a68087e8342f453c80ecf83fb50f9ff6.slice: Unit user-libpod_pod_2e75d2c8edb0d15e52f814f6f11af9c4a68087e8342f453c80ecf83fb50f9ff6.slice already exists.
```

**Describe the results you expected:**

A pod named foo

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
$ podman version
Client:       Podman Engine
Version:      4.3.0-rc1
API Version:  4.3.0-rc1
Go Version:   go1.19.1
Built:        Fri Oct  7 09:31:05 2022
OS/Arch:      linux/amd64
```

**Output of `podman info`:**

```
host:
  arch: amd64
  buildahVersion: 1.28.0-dev
  cgroupControllers:
  - cpu
  - io
  - memory
  - pids
  cgroupManager: systemd
  cgroupVersion: v2
  conmon:
    package: conmon-2.1.4-3.fc37.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.4, commit: '
  cpuUtilization:
    idlePercent: 98.2
    systemPercent: 0.53
    userPercent: 1.27
  cpus: 64
  distribution:
    distribution: fedora
    variant: workstation
    version: "37"
  eventLogger: journald
  hostname: sw-0608
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  kernel: 5.19.14-300.fc37.x86_64
  linkmode: dynamic
  logDriver: journald
  memFree: 244304289792
  memTotal: 270067044352
  networkBackend: netavark
  ociRuntime:
    name: crun
    package: crun-1.6-2.fc37.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 1.6
      commit: 18cf2efbb8feb2b2f20e316520e0fd0b6c41ef4d
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +LIBKRUN +YAJL
  os: linux
  remoteSocket:
    exists: true
    path: /run/user/1000/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: true
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns-1.2.0-8.fc37.x86_64
    version: |-
      slirp4netns version 1.2.0
      commit: 656041d45cfca7a4176f6b7eed9e4fe6c11e8383
      libslirp: 4.7.0
      SLIRP_CONFIG_VERSION_MAX: 4
      libseccomp: 2.5.3
  swapFree: 8589930496
  swapTotal: 8589930496
  uptime: 0h 13m 9.00s
plugins:
  authorization: null
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  volume:
  - local
registries:
  search:
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - docker.io
  - quay.io
store:
  configFile: /home/jdoss/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/jdoss/.local/share/containers/storage
  graphRootAllocated: 998483427328
  graphRootUsed: 713970536448
  graphStatus:
    Backing Filesystem: btrfs
    Native Overlay Diff: "true"
    Supports d_type: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 377
  runRoot: /run/user/1000/containers
  volumePath: /home/jdoss/.local/share/containers/storage/volumes
version:
  APIVersion: 4.3.0-rc1
  Built: 1665153065
  BuiltTime: Fri Oct  7 09:31:05 2022
  GitCommit: ""
  GoVersion: go1.19.1
  Os: linux
  OsArch: linux/amd64
  Version: 4.3.0-rc1
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman` or `brew info podman`):**

```
podman-4.3.0~rc1-3.fc37.x86_64
```

**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide? (https://github.com/containers/podman/blob/main/troubleshooting.md)**

Yes

**Additional environment details (AWS, VirtualBox, physical, etc.):**

This is on Fedora 37 Workstation.

**Comment by Luap99** [comment]
Duplicate of https://github.com/containers/podman/issues/16107
TLDR some bug only present in fedora 37
